.changelog/10611.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:feature
```

Comment:
Not sure if we would generally use `feature` or `improvement` for this kind of thing. I don't have a strange preference either way.